crypto/pkcs11key/pkcs11key_stub.go

```diff
@@ -0,0 +1,5 @@
+// Some Go tools will complain "no buildable Go source files in ..." because
```

Comment:
If we're putting this file in here instead of in #226, the comment from that PR should be ported here.